web/frontend/src/components/navigation/index.svelte

```diff
@@ -1,234 +1,107 @@
-      };
-
-      const posStr = JSON.stringify(position);
-
-      if (knownWaypoints[posStr]) {
-        currentWaypoints[posStr] = knownWaypoints[posStr]!;
-        continue;
-      }
-
-      const marker = createWaypointMarker();
-
-      marker.setLngLat([position.lng, position.lat]).addTo(map);
-
-      currentWaypoints[posStr] = marker;
-      knownWaypoints[posStr] = marker;
-
-      // eslint-disable-next-line no-loop-func
-      marker.getElement().addEventListener('contextmenu', async () => {
-        marker.remove();
-
-        try {
-          await removeWaypoint(client, name, waypoint.getId());
-        } catch (error) {
-          notify.danger((error as ServiceError).message);
-          marker.addTo(map);
-        }
-      });
-    }
-
-    const waypointsToDelete = Object.keys(knownWaypoints).filter(
-      (elem) => !(elem in currentWaypoints)
-    );
-
-    for (const key of waypointsToDelete) {
-      const marker = knownWaypoints[key]!;
-      marker.remove();
-      delete knownWaypoints[key];
-    }
-
-    updateWaypointsId = window.setTimeout(refresh, refreshRate);
-  };
-
-  refresh();
-
-  const robotMarker = new maplibregl.Marker({ color: 'red' });
-
-  robotMarker.setPopup(new maplibregl.Popup().setHTML('robot'));
-
-  const updateLocation = async () => {
-    try {
-      const position = await getLocation(client, name);
-
-      if (!centered) {
-        centered = true;
-        map.setCenter(position);
-      }
-
-      robotMarker.setLngLat([position.lng, position.lat]).addTo(map);
-
-      updateLocationsId = window.setTimeout(updateLocation, refreshRate);
-    } catch (error) {
-      notify.danger((error as ServiceError).message);
-      updateLocationsId = window.setTimeout(updateLocation, refreshRate);
-    }
-  };
-
-  updateLocation();
-};
-
-const handleOpen = () => {
-  initNavigation();
-
-  statusStream?.on('end', () => {
-    clearTimeout(updateWaypointsId);
-    clearTimeout(updateLocationsId);
-  });
-};
-
-const handleClose = () => {
-  clearTimeout(updateWaypointsId);
-  clearTimeout(updateLocationsId);
-};
-
-const handleToggle = (event: CustomEvent<{ open: boolean }>) => {
-  const { open } = event.detail;
-
-  if (open) {
-    handleOpen();
-  } else {
-    handleClose();
-  }
-};
-
 </script>
 
-<Collapse
-  title={name}
-  on:toggle={handleToggle}
->
+<Collapse title={name}>
   <v-breadcrumbs
     slot="title"
     crumbs="navigation"
   />
 
   <div class="flex flex-col gap-2 border border-t-0 border-medium p-4">
-    <v-radio
-      label="Navigation mode"
-      options="Manual, Waypoint"
-      on:input={setNavigationMode}
-    />
-
-    <div
-      id='navigation-map'
-      class="mb-2 h-[400px] w-full"
-    />
+    <div class='flex items-end justify-between'>
+      <div class='flex gap-6'>
+        <div class='flex gap-1 items-end'>
+          <v-input class='w-16' label='Base' tooltip='Specified in lng, lat' readonly value={$robotPosition?.lng} />
```

Comment:
[q] are we manually setting the position of the robot?
Shouldn't we instead use the robot's movementsensor to gather positional and heading data?
[q] how often is the position updated in the case when the robot is navigating to a waypoint.

Reply:
> [q] are we manually setting the position of the robot?

Nope. This is only for monitoring. There's no setting position here. You can set waypoints but that's another part of the UI.

> Shouldn't we instead use the robot's movementsensor to gather positional and heading data?

Good q. I do not know. What I'm using here is what was previously used. @tessavitabile @raybjork happen to know if this method (`navigationService.getLocation()`) is outdated and we should be using another API here?

> [q] how often is the position updated in the case when the robot is navigating to a waypoint.

On average 300ms but varies - this poller will back off and slow down if response times are slow

Reply:
> if this method (navigationService.getLocation()) is outdated and we should be using another API here?

hmmmmmmm, I took at look at the backend for the `getLocation()` method.
I want to say that this method is **not** outdated. 
All we need is the movementsensor the robot is using.
Please note: the movementsensor package is undergoing a refactor so the method signature for `getLocation()` is subject to change.